atef/report.py

```diff
@@ -349,15 +349,18 @@ def build_data_table(
     story.append(Paragraph('Observed Data', l0))
     # use cached value.
     observed_value = getattr(comp, 'data', 'N/A')
-    if not observed_value:
+    if observed_value is None:
```

Comment:
The convenience of omitting `is None` trips me up in these situations, too. I should make more of a point of checking these in reviews

Reply:
It's great to be Pythonic, until it isn't... 😢